hazelcast/src/hazelcast/client/stats.cpp

```diff
@@ -206,12 +206,12 @@ namespace hazelcast {
                 Statistics::PeriodicStatistics::PeriodicStatistics(Statistics &statistics) : statistics_(statistics) {}
 
                 std::string Statistics::escape_special_characters(const std::string &name) {
-                    std::regex reComma(",");
-                    std::string escapedName = std::regex_replace(name, reComma, std::string("\\,"));
-                    std::regex reEqual("=");
-                    escapedName = std::regex_replace(escapedName, reEqual, std::string("\\="));
-                    std::regex reBackslash("\\\\");
-                    escapedName = std::regex_replace(escapedName, reBackslash, std::string("\\\\"));
+                    boost::regex reComma(",");
```

Comment:
If this is the only usage, we probably don't need regex dependency. This part should not be complicated to rewrite without regex.

Reply:
Removed the regex dependency.